src/CTA.WebForms2Blazor/ControlConverters/GridViewControlConverter.cs

```diff
@@ -21,7 +72,32 @@ public GridViewControlConverter() : base()
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            node.OwnerDocument.OptionOutputOriginalCase = true;
+            
+            var itemTypeAttr = node.Attributes.AttributesWithName("itemtype").FirstOrDefault();
+            bool hasItemType = itemTypeAttr != null;
+            var itemType  = itemTypeAttr?.Value ?? "PleaseReplaceWithItemTypeHere";
+
+            IEnumerable<Attribute> newItemTypeAttr = new List<Attribute>()
+            {
+                new Attribute("ItemType", itemType)
+                //"ItemType=" + itemType
+            };
```

Comment:
Can move this code block to a separate function explaining what it does, e.g. `GetItemTypeAttribute()`.

Also, can the commented code be removed?